File: practical_5/fibonacci.c
#include<stdio.h>

int main() {
    int sum = 0, n;
    int a = 0;
    int b = 1;
    
    printf("Enter a number: ");
    scanf("%d", &n);

    printf("The fibonacci series up to %d is: \n", n);

    while(sum <= n) {
        printf("%d ", sum);
        a = b;
        b = sum;
        sum = a + b;
    }

    printf("\n");

    return 0;
}

File: practical_5/fibonacci.py
n = int(input("Enter a number: "))

a = 0
b = 1
sum = 0

print("The fibonacci series up to", n, "is:")

while(sum <= n):
     print(sum, end = " ")
     a = b
     b = sum
     sum = a + b

print()
